config/sara/jurisdiction_messages.yml

```diff
@@ -19,11 +19,13 @@
             sms:
               prompt:
                 intro:
+                  # For jurisdictions that use a custom intro SMS message, the privacy policy info will automatically be appended to the custom message
+                  # if this link is not already included in the message
                   eng: |
                     You've been identified by State 1 DOH to be enrolled in Sara Alert to be monitored for COVID symptoms.
 
-                    For privacy and more info, visit saraalert.org.
+                    For privacy and more info, visit example.org.
```

Comment:
iirc I chose this approach to avoid having to modify our prod jurisdiction messages but I think you're right in that it'll simplify things in the long term

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1247/commits/8fac5befa1faba9061b8ef057506327d991f57f2